source/tutorial/install-mongodb-on-windows.txt

```diff
@@ -44,6 +44,16 @@ enter the following command in either the :guilabel:`Command Prompt` or
    OS Version:                10.0.17134 N/A Build 17134
    System Type:               x64-based PC
 
+Make certain that you show set Windows Explorer to show file name 
+extensions for all file types. This can prevent issues where the file 
```

Comment:
I think this is closer to:

"This can prevent issues where the file type displayed to the user differs from the actual file type."